docs/core/tools/dotnet-nuget-sign.md

```diff
@@ -0,0 +1,162 @@
+
+- **`--certificate-subject-name <SUBJECTNAME>`**
+
+  Specifies the subject name of the certificate used to search a local certificate store for the certificate. The search is a case-insensitive string comparison using the supplied value, which will find all certificates with the subject name containing that string, regardless of other subject values. The certificate store can be specified by --certificate-store-name and --certificate-store-location options.
+
+> [!NOTE]
+> This command currently supports only a single matching certificate in the result. If there are multiple matching certificates in the result, or no matching certificate in the result, the command will fail.
+
+- **`--certificate-fingerprint <FINGERPRINT>`**
+
+   SHA-1 fingerprint of the certificate used to search a local certificate store for the certificate.
+
+- **`--certificate-password <PASSWORD>`**
+
+   Specifies the certificate password, if needed. If a certificate is password protected but no password is provided, the command will fail.
+
+> [!NOTE]
+> This command currently supports only noninteractive mode, so there won't be any prompt for a password at run time.
+
+- **`--hash-algorithm <HASHALGORITHM>`**
+
+   Hash algorithm to be used to sign the package. Defaults to SHA256. Possible values are SHA256, SHA384, and SHA512.
+
+- **`--timestamper <TIMESTAMPINGSERVER>`**
+
+   URL to an RFC 3161 timestamping server.
+
+- **`--timestamp-hash-algorithm <HASHALGORITHM>`**
+
+   Hash algorithm to be used by the RFC 3161 timestamp server. Defaults to SHA256.
+
+- **`--overwrite`**
+
+   Switch to indicate if the current signature should be overwritten. By default the command will fail if the package already has a signature.
+
+* **`-v|--verbosity <LEVEL>`**
+
+  Sets the [MSBuild verbosity level](/visualstudio/msbuild/obtaining-build-logs-with-msbuild#verbosity-settings). Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`. The default is `minimal`.
+
+* **`-h|--help`**
+
+  Prints out a short help for the command.
+
+## Examples
+
+- Sign *foo.nupkg* with certificate *cert.pfx* (with no password protected):
+
+  ```dotnetcli
+  dotnet nuget sign foo.nupkg --certificate-path cert.pfx
+  ```
+
+- Sign *foo.nupkg* with certificate *cert.pfx* (with password protected):
+
+  ```dotnetcli
+  dotnet nuget sign foo.nupkg --certificate-path cert.pfx --certificate-password password
+  ```
+
+- Sign *foo.nupkg* with certificate(with password protected) matches with the specified SHA-1 fingerprint in the default certificate store(CurrentUser\My):
+
+  ```dotnetcli
+  dotnet nuget sign foo.nupkg --certificate-fingerprint 89967D1DD995010B6C66AE24FF8E66885E6E03A8 --certificate-password password
+  ```
+
+- Sign *foo.nupkg* with certificate(with password protected) matches with the specified subject name *Test certificate for testing signing* in the default certificate store(CurrentUser\My):
```

Comment:
```suggestion
- Sign *foo.nupkg* with certificate (with password protected) matches with the specified subject name *Test certificate for testing signing* in the default certificate store (CurrentUser\My):
```

Reply:
"Test certificate for testing signing" should be in a no-loc.

Reply:
Thanks! Could you pls point me to an example of no-loc comment?

Reply:
Thanks! No-loc comment is added for "Test certificate for testing signing".